contents/handbook/company/management.md

```diff
@@ -6,7 +6,13 @@ showTitle: true
 
 As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
 
-## Defining the role of manager
```

Comment:
Either "defining the role of managers" of "of a manager"

Reply:
Ah I think you're looking at the old diff here? I did change this.